pupil-api/src/services/redis-pupil-auth.service.spec.ts

```diff
@@ -79,6 +85,46 @@ describe('redis-pupil-auth.service', () => {
     expect(payload).toEqual(expectedPayload)
   })
 
+  test('the check payload should not be returned if check pinValidFromUtc timestamp is after current date', async () => {
```

Comment:
Perhaps you could rephrase this test to save us all a headache 🤕 😁 

Suggest 'authorisation is denied if the pin is not yet valid'?